docs/reference/search.md

```diff
@@ -0,0 +1,99 @@
+# Searching in InvenioRDM
+
+### InvenioRDM Suggest API
+
+The suggest API endpoint (`/api/{resource}/?suggest={search_input}`) provides an interface for real-time search suggestions. It leverages OpenSearch's `multi_match` query to search across multiple fields within a specified index, returning relevant suggestions based on user input.
+
+### Endpoint Structure
+
+**URL:** `/api/{index-name}?suggest={search_input}`
```

Comment:
Is there a difference between `index-name` and `resource`? `index-name` sounds like the internal index name (e.g. `zenodo-qa-names-name-v1.0.0-1708098159`). We should be consistent with the terminology use.